launch/docker_test.go

```diff
@@ -216,6 +228,186 @@ func TestRunBuildWithSudo(t *testing.T) {
 	}
 }
 
+func TestDockerKill(t *testing.T) {
+	t.Run("success to kill process with no commands", func(t *testing.T) {
```

Comment:
every test case use words of `to kill process`. It looks redundancy.
```suggestion
	t.Run("success with no commands", func(t *testing.T) {
```